### ISSUE TITLE
quotes do not keep word together

### ISSUE THREAD

**Issue by npnorman** [Eagle-9/crash#92]
From Project 2

A word is a sequence of characters considered as a single unit by the program. Words are separated by a
metacharacter if unquoted. The metacharacters includes
| & ; ( ) < > space tab newline

this is not currently happening
example
```
crash $ if cd ..; then echo "Hello world"; endif
```

my tokens look as so if I print them out
```
if
cd
..
then
echo
"Hello
world"
endif
```

should this not be:
```
if
cd
..
then
echo
"Hello world"
endif
```

**Comment by Derp88** [comment]
yeah this is an issue
I'll try to get it fixed tomorrow

**Comment by Derp88** [comment]
Alright I believe I have fixed this in commit d4596ee in [branch 84](https://github.com/Eagle-9/crash/tree/84-pipe-and-redirection)

Using the same test expression `if cd ..; then echo "Hello world"; endif`

I get
```
Token: 0 keyword Data: if
Token: 1 internal Data: 
Token: 2 argument Data: ..;
Token: 3 keyword Data: then
Token: 4 external Data: echo
Token: 5 argument Data: "Hello world";
Token: 6 keyword Data: endif
```

The semicolons are still there as they are not handled yet.
